src/features/web-compat.js

```diff
@@ -72,6 +72,29 @@ function permissionsFix (settings) {
     }
 }
 
+function messageHandlersFix () {
+    // Do nothing if `messageHandlers` is absent
+    if (!window.webkit?.messageHandlers) return
```

Comment:
probably should move to using `globalThis` always for node and keep @sammacbeth happy :D